scanpy/tests/test_embedding_plots.py

```diff
@@ -115,8 +115,8 @@ def groupsfunc(request):
 
 
 @pytest.fixture(
-    params=[(None, None), (0, 5), ("p15", "p90")],
-    ids=["vbounds.default", "vbound.numbers", "vbound.percentile"],
+    params=[(None, None, None), (0, 5, None), ("p15", "p90", None), (0, "p99", 0.1)],
+    ids=["vbounds.default", "vbound.numbers", "vbound.percentile", "vbound.vcenter"],
 )
 def vbounds(request):
     return request.param
```

Comment:
Could this be a bit more explicit about the values? This is mostly an ease of reading thing. Here's how I'd do it:



```suggestion
    params=[
        pytest.param(
            {"vmin": None, "vmax": None, "vcenter": None},
            id="vbounds.default"
        ),
        ...  # Fill in the rest
)
def vbounds(request):
    return request.param
```

It's more text, but it's very easy to tell what happening. It also allows just splatting the parameter into the call, or `update`-ing the parameter `dict`.

What do you think about also including `norm` into this `fixture`?

Reply:
Done. Added norm too.